action.yml

```diff
@@ -514,8 +514,15 @@ runs:
           -n ${{ inputs.containerAppName }} \
           --yaml ${{ inputs.yamlConfigPath }}
 
+    - name: Determine whether or not 'update' or 'up' should be used
```

Comment:
On the long run, should this logic be done in the CLI? If yes, then should we have a work item there so that on the long run the situation is improved globally, even though in the meantime this fix will help users.

Reply:
I agree that, ideally, we wouldn't need to make this kind of distinction in our tooling -- the core issue is the lack of updating ingress settings through the CLI, as `az containerapp ingress enable` performs a `PUT` operation that wipes all extraneous ingress properties, and there doesn't exist an `az containerapp ingress update` command.

Unless there's another place we should open the work item, I'll open an issue in the extensions GitHub repository that requests an `ingress update` command, as well as clarifies the `ingress enable` documentation that it does not _update_ the ingress if already active.

Reply:
Yeah it would be nice if `ingress update` is part of ACA plans.

Side note:
Alternatively, we probably can leverage CLI sub-arguments in bash. However, I have found Python's to be the cleanest syntax IMO: https://stackoverflow.com/questions/10448200/how-to-parse-multiple-nested-sub-commands-using-python-argparse